entrypoint.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 set -o pipefail
 
-/format.jl "$@"
+julia --project=$GITHUB_WORKSPACE format.jl "$@"
```

Comment:
We don't need to do --project. If anything I think this may potentially be harmful since when we add JuliaFormatter it will mutate the Project.toml file. I'm not sure if the change is saved regardless but removing `--project` avoids the situation entirely.